### PR TITLE
:robot: Add recaf.vm

### PR DESCRIPTION
**Pull request by vm-packages**
Automated PR from https://github.com/mandiant/VM-Packages/issues/1155 :rocket:

### CODE REVIEW

File: packages/recaf.vm/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'recaf-2.21.14-J8-jar-with-dependencies.jar'
```
Comment (Ana06) [inline packages/recaf.vm/tools/chocolateyinstall.ps1:4]: The tool name should not include the `.jar`, as this makes this look different than other tools:
image

The issue template mentions the name should not include the `.exe`, not sure if we should rephrase it to make it clear for other cases (like the one we have here with `.jar`:
>The name of the tool being installed (usually the file name with the .exe), normally different from the package name. Example: FakeNet-NG (tool name) vs fakenet-ng (package name).

Please remove the `.jar`

**Review by Ana06 — CHANGES_REQUESTED** [review]
Thanks for adding the dependency @sara-rn, the tool works well now, but the name is incorrect, causing it to show the `.jar` in the shortcut and a duplication `.jar.jar` in the tool name. Can you please fix that? :pray:

File: packages/recaf.vm/tools/chocolateyinstall.ps1 (hunk 1 above)
Comment (sara-rn) [reply packages/recaf.vm/tools/chocolateyinstall.ps1]: its fixed

**Review by Ana06 — COMMENTED** [review]
@sara-rn please squash your commits in order to keep a clean commit history. :smile:

**Review by Ana06 — COMMENTED** [review]
@sara-rn similarly as in https://github.com/mandiant/VM-Packages/pull/1162#pullrequestreview-2454057986,  can you please reword the commit message to explain why the change is needed without included the squashed `fixup` commit messages? For example:
```
Add recaf.vm

Add Java bytecode editor useful for RE.
```

**Comment by sara-rn** [comment]
Closes https://github.com/mandiant/VM-Packages/issues/1155
